x-pack/legacy/plugins/reporting/export_types/png/server/create_job/index.ts

```diff
@@ -4,24 +4,25 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { Request } from 'hapi';
+import { KbnServer, ConditionalHeaders } from '../../../../types';
 import { cryptoFactory } from '../../../../server/lib/crypto';
 import { oncePerServer } from '../../../../server/lib/once_per_server';
+import { JobParamsPNG as JobParams, CreateJobFactoryPNG as CreateJobFactory } from '../../types';
```

Comment:
Note how I've had to make specific types just for the PNG export_type - I think we'll have to do that with all of the export_types.

The `JobParams` that's in the main types file right now might be the CSV-From-Saved-Object format of job params.